README.md

```diff
@@ -1,2 +1,68 @@
-# .github
-Organization information
+# Hydra-Genetics
+
+We are an organization/community with the goal of making [snakemake](https://snakemake.readthedocs.io/en/stable/index.html) pipeline development easier and faster, and a bit more structured. 
+
+We do this by providing [snakemake modules](https://snakemake.readthedocs.io/en/stable/snakefiles/modularization.html#modules) that can be combined to create a complete analysis or included in already existing pipelines. All modules are subjected to extensive testing to make sure that new releases doesn't unexpectedly break existing pipeline or deviate from guidelines and best practices on how to write code. 
+
+Current tests:
+- [pycodestyle](https://pycodestyle.pycqa.org/en/latest/): (--max-line-length 130)
+- [snakefmt](https://github.com/snakemake/snakefmt): (line-length 130)
+- [snakemake](https://snakemake.readthedocs.io/en/stable/executing/cli.html?highlight=lint#UTILITIES) --lint: (line-length 130)
+- [snakemake dry-run](https://snakemake.readthedocs.io/en/stable/executing/cli.html#useful-command-line-arguments)
+- execution test with small dataset
+- ~~execution: integration test witt complete dataset~~: planned to be implemented 
+
+The small execution tests will make sure that the pipeline actually can be executed, i.e run and generate data. The integration test with complete data set will also evaluate the generated result to make sure that results does not change, if it's not deliberately.  
+
+We also have a [development document](https://docs.google.com/document/d/1l2v1ItZBTDaI72vQPZcaQzxwVUao78XzIJvGASAAD9E/edit?usp=sharing) describing how a hydra-genetic pipeline/module should be structured, named and what a rule should contain.    
+
+## Wiki
+Please visit our [wiki](https://github.com/hydra-genetics/.github/wiki/Welcome-to-hydra-genetics) for more information about using hydra-genetics.
+
+## Repositories 
+The current repositories can be divided into the following sections.
+
+### Tools and examples
+
+#### [hydra-genetics/tools](https://github.com/hydra-genetics/tools)
+Command line interface to create new modules/pipelines or adding a new rule to a existing project. Provides libraries used to make it easier for people not used to [pandas](https://pandas.pydata.org/) to extract information from samples and units dataframes, these dataframes are generated from [units.tsv](https://github.com/hydra-genetics/prealignment/blob/develop/workflow/schemas/units.schema.yaml) and [samples.tsv](https://github.com/hydra-genetics/prealignment/blob/develop/workflow/schemas/samples.schema.yaml) files  and used as input for hydra-genetics. 
```

Comment:
Feels a bit misleading to link to units.tsv and then get a schema. Instead link to example and the schema.